README.md

```diff
@@ -1 +1,128 @@
-# capstone-project-1
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+  <h3><b>Microverse</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🚀 Visit Site](#website)
+  - [🎥 Walkthrough](#walkthrough)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Capstone Project 1] <a name="about-project"></a>
+
+This project is the capstone project that portrays a conference website.
+
+**[Capstone Project]** is a simple project demonstrating responsive conference website.
+
+## 🚀 VISIT SITE <a name="website"></a>
+
+<a name="<https://king-benx.github.io/my-portfolio/>">https://king-benx.github.io/my-portfolio/</a>
+
+## 🎥 WALKTHROUGH <a name="walkthrough"></a>
+
+<a name="<<https://www.loom.com/share/785d791fbd164d8b9a4ad7f206cc0393/>>">https://www.loom.com/share/785d791fbd164d8b9a4ad7f206cc0393/</a>
```

Comment:
- [ ] **Good job on recording a ```loom video``` presenting your project 👍 👍**, However, It would be better to highlight a piece of the code that finds interesting or that you are proud of making. 😂 

    As this can help to boost confidence and motivation by recognizing your accomplishments, it would also make your presentation video more professional 👍 

    **KEEP IT UP 😉**